lib/cast.rb

```diff
@@ -0,0 +1,20 @@
+class Cast
```

Comment:
Fair enough, shall I move this to the foreman directory as well as adding it to Foreman's namespace?